### DISCUSSION TITLE
Inconsistent search results issue - do you have it too?

### DISCUSSION THREAD

**Discussion by LetMeSP** [microsoft-search/pnp-modern-search#1998] [Q&A] [answered]
In the last few weeks I observed the following random behavior in the PnP search results web part (search query to include a specific content type only):

- The shown number of results (displayed on top) can change on each page refresh (for example I am expecting 12 results but when refreshing it shows number 11 or number 9)
- The shown number of documents is different from the number of results on top (for example it shows number 12 on top but only 11 document previous are shown; after refresh it might again show it correctly)

I believe this is a Microsoft issue as I see a this behavior with Powell's search web parts as well. I already opened a ticket with them but just wanted to check if you see a similar behavior in your tenants.

**Comment by patrikhellgren** [comment, marked as answer]
Yes, we have seen the same issue in multiple tenants and it most certainly has to do with the issue reported in the Service Health dashboard [Users may see inconsistent results when searching within SharePoint Online](https://admin.microsoft.com/Adminportal/Home#/servicehealth/history/:/alerts/SP349177).
It was reported as resolved a few hours ago but I haven't tested yet.
